test/tests/functional/pbs_multi_sched.py

```diff
@@ -580,6 +580,60 @@ def test_run_limts_per_scheduler(self):
         jc = "Not Running: User has reached server running job limit."
         self.server.expect(JOB, {'comment': jc}, id=jid4)
 
+    def test_list_multi_sched(self):
+        """
+        Test to verify that qmgr list sched works when multiple schedulers are present
+        """
+
+        self.setup_sc1()
+        self.setup_sc2()
+        self.setup_sc3()
+
+        self.server.manager(MGR_CMD_LIST, SCHED, expect=True)
+
```

Comment:
why the blank lines between manager calls? PEP8 says "Use blank lines in functions, sparingly, to indicate logical sections."

Reply:
blank lines are used sparingly, since each manager call executes the targeted test objective, I consider each of them equivalent to a logical section.